api/Element.json

```diff
@@ -5189,7 +5209,7 @@
             ],
             "edge": [
               {
-                "version_added": "≤18"
+                "version_added": "15"
               },
               {
                 "version_added": "≤18",
```

Comment:
I've confirmed using Sauce that `webkitMatchesSelector` was present in Edge 13. It was probably in Edge 12, but I don't have my computer with the Edge 12 on it. Would be good to fix this too, but not necessary in this PR.

Reply:
It looks like it's present in Edge 12 as well -- patch coming right up!